dist/tools/buildsystem_sanity_check/check.sh

```diff
@@ -415,13 +415,14 @@ all_checks() {
 
 main() {
```

Comment:
maybe rename this function while you're at it - the C developer might think this is the entry point of the script :wink: